lib/services/screen-builder-service.ts

```diff
@@ -5,18 +5,28 @@ import * as path from "path";
 import * as util from "util";
 
 export class ScreenBuilderService implements IScreenBuilderService {
+	private static SCREEN_BUILDER_SPECIFIC_FILES = [".yo-rc.json", ".app.json", "app.js"];
+	private static UPGRADE_ERROR_MESSAGE = 'Your app has been build with an obsolete version of Screen Builder. Please, migrate it to latest version and retry.';
+	// This constant is introduced due to the original message (the one above)
+	// not being descriptive enough.
+	// It should be removed once Screen Builder team fixes the message from their side in the next version.
+	private static UPGRADE_ERROR_MESSAGE_SHOWN_ON_THE_CONSOLE = 'Your app has been created with an obsolete version of Screen Builder. You need to upgrade your to be able to run Screen Builder-related commands.';
```

Comment:
yeah, I have missed a "project" in there. Sorry about that. It must be "You need to upgrade your to be able to run Screen Builder-related commands."